### DISCUSSION TITLE
M.I.B charge FECS for update maps

### DISCUSSION THREAD

**Discussion by ldc1966** [Mr-MIBonk/M.I.B._More-Incredible-Bash#258] [General]
Hi, 
I uploaded the MIB V3.5.0 "Spring Edition" on my AUDI MHI2Q_ER_AUG22_P4212 device with the corresponding patch. 
I have activated GEM however I cannot load the FEC 023000EE which allows map update
This is the log file :
[MU1177-JTB-07407.05.18119202DA.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11806262/MU1177-JTB-07407.05.18119202DA.log)

I activated the original + android dual navigation and it works.

Previously I had entered the personal POI version.it_IT=Version: 2023-02-24 , this could be the problem , how to solve it ?

I have to eliminate the SVM error, can this affect it?

**Comment by harman-f** [comment]
Use 3.6 version
Patch can not be found on SD card

**Reply by ldc1966** [reply]
![Cattura1](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/137117054/3d9ec316-ab5e-4224-8abc-f2b4027d1fd4)
![Cattura2](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/137117054/99b04404-6e23-49bc-a00e-570863ce4a84)

**Reply by ldc1966** [reply]
This is correct ?

**Comment by ldc1966** [comment]
Greetings, thanks a lot for the support. I will upload the V3.6.0 "QR
Edition" version.
Can you tell me if the file structure is correct? Is it possible to enable
google view with MIB ?



Il Mar 20 Giu 2023, 23:17 harman-f ***@***.***> ha scritto:

**Comment by ldc1966** [comment]
Found the solution I had copied the contents of the folder and not the folder in the "patch" folder of the SD.
Thank you
